packages/docs/content/docs/concepts/cli.mdx

```diff
@@ -127,6 +127,13 @@ Options:
 
 - `-d, --dir <step file path>`: The path relative to the steps directory to create the step file.
 
+#### `generate openapi`
+
+Generate OpenAPI spec for your project.
+
+```bash
+npx motia generate openapi
+```
```

Comment:
can you add the list of options available?

i noticed you added title and version but we should also add the output file